fix: some typos in mimi-content.md (#41)
ietf-wg-mimi/draft-ietf-mimi-content: draft-ietf-mimi-content.md

Joining a conference via an external URL is possible. The link could be
rendered to the user, requiring a click. Alternatively the URL could be

> rendered the
> disposition could be specified as `session` which could be processed

Reviewer: ```suggestion
in an ExternalPart with a disposition of `session`, which could be processed
```
